src/app/views/query-runner/query-input/AutoComplete.tsx

```diff
@@ -0,0 +1,264 @@
+import { getTheme, KeyCodes, Label, TextField } from 'office-ui-fabric-react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { IAutoCompleteProps, IAutoCompleteState } from '../../../../types/auto-complete';
```

Comment:
Separate local imports and  package imports with a new line.

Reply:
good catch